roles/lnls-ans-role-sirius-hlacons/tasks/clone-install-app.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: "Ensure repository {{ item.repo_name }} is cloned"
+  git:
+    repo: "{{ item.org_url }}/{{ item.repo_name }}"
+    dest: "{{ item.clone_path }}/{{ item.repo_name }}"
+    version: "{{ item.repo_version | default('master') }}"
+    accept_hostkey: "{{ item.git_accept_hostkey | default(omit) }}"
+    ssh_opts: "{{ item.git_ssh_opts | default(omit) }}"
+    force: "{{ item.force_version | default(false) }}"
+
+- name: "Install app at cons Conda env"
+  shell:
```

Comment:
Could we use `command` module for this? It's generally a safer alternative then `shell`, as explained here: https://docs.ansible.com/ansible/latest/modules/command_module.html